### ISSUE TITLE
 WARNING [139954094954240] (ds5-timestamp.cpp:64) UVC metadata payloads not available. Please refer to installation chapter for details

### ISSUE THREAD

**Issue by weiwuhuhu** [realsenseai/realsense-ros#492]
hello,
     when i execute the roslauch realsense2_camera rs_rgbd.launch ,the warning has occurred,

             11/10 22:03:53,863 WARNING [139954094954240] (ds5-timestamp.cpp:64) UVC metadata                                    payloads not available. Please refer to installation chapter for details.
             [ WARN] [1539266633.868420728]: Frame metadata isn't available! (frame_timestamp_domain = RS2_TIMESTAMP_DOMAIN_SYSTEM_TIME).
            What's influence of this warning.

**Comment by dorodnic** [comment]
Hi @weiwuhuhu 
If you follow [distribution_linux.md](https://github.com/IntelRealSense/librealsense/blob/master/doc/distribution_linux.md) the following command should solve the problem:
`sudo apt-get install librealsense2-dkms`

**Comment by RealSenseCustomerSupport** [comment]
[Please Ignore - RealSense system comment]

**Comment by RealSenseCustomerSupport** [comment]
----------------------------------------------

Close this issue for lack of activity. Please feel free to reopen if the issue is still relevant.

**Comment by SuperKuooo** [comment]
Hi, I am still having this problem. I am running ROS melodic on Ubuntu 18.04. I ran the command given by @dorodnic. Please help! Thanks.

**Comment by jarvis-owl** [comment]
Hej @SuperKuooo ,
if you still care:  the topics are being published though.
You can watch the stream with `rqt_image_view`

**Comment by BriceRenaudeau** [comment]
Hello, I hade the same problem :
WARNING [...] (ds5-timestamp.cpp:64) UVC metadata payloads not available
[ WARN] [/camera_right/realsense2_camera_manager]: Frame metadata isn't available! (frame_timestamp_domain = RS2_TIMESTAMP_DOMAIN_SYSTEM_TIME)

[Ubuntu 18.04, kernel 5.0, melodic, D435, firmware 5.12, librealsense2-dkms up-to-date]

I hade no issues until today where I found that the TimeStamp in the depth images published by my camera where stuck in the past.
The stamps were oscillating in the past (not fixed).
Strangely when I added `<arg name="filters" default="pointcloud"/>` in the launch file, the stamps were good again.
